ibm/service/globaltagging/data_source_ibm_resource_tag.go

```diff
@@ -60,6 +67,12 @@ func dataSourceIBMResourceTagRead(d *schema.ResourceData, meta interface{}) erro
 		tType = t.(string)
 	}
 
+	tReplace := false
```

Comment:
why are we reading from user input and setting back the same value.
This attribute is not required for datasource